device-types/Cisco/N3K-C3172PQ-10GE.yaml

```
@@ -1,14 +1,18 @@
 ---
 manufacturer: Cisco
-model: Nexus 3172PQ-10GE
-slug: cisco-n3k-c3172q-10ge
-part_number: N3K-C3172Q-10GE
+model: Nexus C3172PQ-10GE
```

```suggestion
model: Nexus 3172PQ-10GE
```
In keeping with all of the other Nexus switches, the "C" should not be part of the model name.

---

"C" removed from model name